_topic_maps/_topic_map.yml

```diff
@@ -2357,6 +2357,8 @@ Topics:
   - Name: Installing and configuring OADP
     Dir: installing
     Topics:
+    - Name: About installing OADP
```

Comment:
@pneedle-rh I don't recall making this change. Should I remove it? Is it causing my build to break?

Reply:
@RichardHoch hi! In the `_topic_maps/_topic_map.yml` in your feature branch, these two lines are duplicated. This is causing the build failure.

Please remove the two duplicate lines and see if this resolves the issue.